images/win/scripts/Installers/Install-SeleniumWebDrivers.ps1

```diff
@@ -2,16 +2,45 @@
 ##  File:  Install-SeleniumWebDrivers.ps1
 ##  Desc:  Install Selenium Web Drivers
 ################################################################################
+$DestinationPath = "C:\";
+Write-Debug "Destination path: [$DestinationPath]";
+Write-Debug "Selenium drivers download and install...";
+Invoke-WebRequest -UseBasicParsing -Uri "https://seleniumwebdrivers.blob.core.windows.net/seleniumwebdrivers/SeleniumWebDrivers.zip" -OutFile SeleniumWebDrivers.zip;
+Expand-Archive -Path SeleniumWebDrivers.zip -DestinationPath $DestinationPath -Force;
+Remove-Item SeleniumWebDrivers.zip;
```

Comment:
I think we can create a variable for archive name to store it in a single place, something like `$DestinationFile = "SeleniumWebDrivers.zip"` and then use it

Reply:
thanks, done